diff --git a/src/partition_alloc/pointers/raw_ptr.h b/src/partition_alloc/pointers/raw_ptr.h
index c3ac1152..2a3d3bc4 100644
--- a/src/partition_alloc/pointers/raw_ptr.h
+++ b/src/partition_alloc/pointers/raw_ptr.h
@@ -1271,7 +1271,7 @@ template <typename T,
           typename TQ,
           template <typename>
           typename UQ>
-struct std::basic_common_reference<raw_ptr<T, Traits>, T*, TQ, UQ> {
+struct basic_common_reference<raw_ptr<T, Traits>, T*, TQ, UQ> {
   using type = T*;
 };
 
@@ -1281,7 +1281,7 @@ template <typename T,
           typename TQ,
           template <typename>
           typename UQ>
-struct std::basic_common_reference<T*, raw_ptr<T, Traits>, TQ, UQ> {
+struct basic_common_reference<T*, raw_ptr<T, Traits>, TQ, UQ> {
   using type = T*;
 };
 
